packages/encryption/tests/encryption.ts

```diff
@@ -28,9 +28,10 @@ const publicTarget: EncryptOptions['target'] = {
 }
 const example: EncryptOptions = {
     version: -38,
-    author: new ProfileIdentifier('localhost', 'alice'),
+    author: ProfileIdentifier.of('localhost', 'alice').unwrap(),
     message: makeTypedMessageText('hello world'),
     target: publicTarget,
+    network: 'localhost',
```

Comment:
Can we replace the string with `EnhanceableSite.Localhost`？

Reply:
No. It is just a test file.